pyairtable/orm/model.py

```diff
@@ -46,6 +47,7 @@ class Meta:
                 api_key = "keyapikey"
                 timeout = (5, 5)
                 typecast = True
+                use_field_ids = True
```

Comment:
Let's leave this out of the example, since the sample code (above) uses field names and not IDs. We want the example to remain internally consistent.